config/initializers/active_record_enum.rb

```diff
@@ -0,0 +1,11 @@
+module ActiveRecord
+  module Enum
```

Comment:
We are not using this anymore, right? You can delete it

Reply:
Actually I am, here's a reference as to why I did that https://github.com/rails/rails/issues/13971

Reply:
Miss that!
As Bruno , I have never seen this before. But, if you are confortable with this, go ahead!